### PR TITLE
April 2017 meeting

### PR DESCRIPTION
**Pull request by tobyhodges**
Minutes of April 2017 meetings.

### CODE REVIEW

**Comment by tobyhodges** [comment]
No water spillage-related delays this month, thankfully!

File: subcommittees/mentoring/minutes/minutes-2017-04-24.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+http://pad.software-carpentry.org/instructor-discussion
+
+###### First Meeting
+
+- A lot of the activity of the mentoring subcommittee over the last year has been focussed on organising and developing the discussion sessions ( http://pad.software-carpentry.org/instructor-discussion. Part of our future efforts will be encouraging people to continue to to get involved in these sessions as a host/co-host. and raise awareness to participate in sessions in general.
+- Will need new hosts soon for May sessions
+- Do you know anyone who may be interested n co-hosting/not-taking with a more experienced discussion leader?
+  - In the last meeting, we discussed the idea of pairing experienced and novice hosts/co-hosts/notetakers
+
+CH: about to lead my first session. Wants to avoid running through the same questions with everyone, to try to maintain organic flow of the sessions. What really ehlped me prepare was being a note-taker a few sessions becoming a host
```
Comment (mkuzak) [inline subcommittees/mentoring/minutes/minutes-2017-04-24.md]: ehlped > helped

File: subcommittees/mentoring/minutes/minutes-2017-04-24.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+
+JS: Perhaps include discussion session signup as part of the instructor training, to improve rate. +1 Could also ask for a host, co-host/notetaker, and a 'veteran', to ensure that there is always sufficient experience on the calls.
+
+TM: need to think carefully about what the limits should be. Dont want to remove the incentive for people checking out. 
+
+JH: There's also been some discussion about what a checking out instructor trainee can do , if the times/dates of the available discussion sessions are all inconvenient/inaccessible for them.
+
+MK: to avoid the problem of not allowing people the opportunity to checkout, it would be really good to have some estimate of the number of people that we are expecting. Can we get those numbers? Havinga  liason between this group and the instructors trainers would be very helpful for this. It's good to observe several different hosts before leading your own sessions - different hosts have different styles and it's good to see the different ways of doing things.
```
Comment (mkuzak) [inline subcommittees/mentoring/minutes/minutes-2017-04-24.md]: Havinga > having a

**Comment by tobyhodges** [comment]
changes made

**Pull request merged**